test/quantization/core/test_quantized_op.py

```diff
@@ -4795,7 +4798,7 @@ def test_qconv2d(
            Y_scale=st.floats(4.2, 5.6),
            Y_zero_point=st.integers(0, 4),
            use_bias=st.booleans(),
-           post_op=st.sampled_from(["add"]),
+           post_op=st.sampled_from(["add", "add_relu"]),
```

Comment:
I think it might be better to duplicate the code so that each test is only testing one op

Reply:
Thanks for the comment. Split "add" and "add_relu" into separate testing so each test is only testing one op.